code/WorkInProgress/Cael_Aislinn/sculpture.dm

```diff
@@ -291,21 +292,3 @@
 
 //You cannot destroy SCP-173, fool!
 /mob/living/simple_animal/sculpture/ex_act(var/severity)
-
-// Commented this out because it LITERALLY PREVENTS ADMINS FROM REMOVING IT ENTIRELY
-//But seriously, you just can't
-// /mob/living/simple_animal/sculpture/Del()
-//
-// 	replace_SCP()
```

Comment:
Would still love to know which absolute madman decided this was a good idea, without even providing an off switch.